CONTRIBUTING.md

```diff
@@ -3,6 +3,52 @@
 1. Create empty package: `npx lerna create @cdk-seed/<package-name>`
 1. Prepare package: `npx jsii-config-p ./packages/<package-name>/package.json`
 
+All TypeScript tooling is managed by [`jsii`](https://github.com/aws/jsii) and generated by running `npm build`. There is no need to add or modify TypeScript build configuration (e.g. `tsconfig.json`).
+
+## Documentation
+
+To support others who want to use our distributed constructs, we want to ensure that each package distributed by CDK Seed is well documented.
+
+### README
+
+Each package should contain a `README.md` at its root. We use [Github Flavored Markdown](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) to render the content into HTML. The README should be formatted as follows:
+
+```markdown
+# <!--Construct Name -->
+
+<!-- A brief description of what the construct's purpose -->
+
+## Use Cases
+
+<!-- Some examples of when a user may want to utilize this construct -->
+
+## Examples
+
+<!-- Examples of how to use the construct -->
+
+## Requirements
+
+<!-- Expectations of user's VPC configuration, etc -->
+```
+
+Code samples can be included alongside text within in markdown files as such:
+
+    <div class="code-example">
+
+    Below is an example of doing something with code!
+
+    </div>
+
+    ```ts
+    console.log("Look, I'm doing something!")
+    ```
+
+For more UI components, see the [Just the Docs](https://pmarsceill.github.io/just-the-docs/docs/ui-components) documentation. _NOTE: Just the Docs' examples utilize Kramdown to render its markdown, not Github Flavored Markdown. Being that we use Github Flavored Markdown, shorthand such as `{: .btn }` is not available. Any non-standard markdown should be written as HTML._
```

Comment:
What do you think about adding a newline before the `_NOTE` section?

Reply:
```suggestion
For more UI components, see the [Just the Docs](https://pmarsceill.github.io/just-the-docs/docs/ui-components) documentation.

_NOTE: Just the Docs' examples utilize Kramdown to render its markdown, not Github Flavored Markdown. Being that we use Github Flavored Markdown, shorthand such as `{: .btn }` is not available. Any non-standard markdown should be written as HTML._
```